README.md

```diff
@@ -74,18 +62,10 @@ let EddRequestability = bySierraLocation['mal']['recapLocation']['eddRequestable
 
 For a comprehensive list of availability see the implementation of factories mentioned `nypl-core-objects.js`
 
-### Supported Object Types
-
-* Locations (from [locations.json](https://github.com/NYPL/nypl-core/blob/master/vocabularies/json-ld/locations.json) & [recapCustomerCodes.json](https://github.com/NYPL/nypl-core/blob/master/vocabularies/json-ld/recapCustomerCodes.json))
-
-* Patron Types (from [patronTypes.json](https://github.com/NYPL/nypl-core/blob/master/vocabularies/json-ld/patronTypes.json))
-
-* Catalog Item Types (from [catalogItemType.json](https://github.com/NYPL/nypl-core/blob/master/vocabularies/json-ld/catalogItemTypes.json))
-
 ## Git Workflow
 
 This repo has two target branches:
- - `master` for Node14+ support (module versions 2x)
+ - `master` for Node14+ support (module versions 3x)
```

Comment:
Are there still node 6 apps that depend on this repo?

Reply:
Yes. The discovery-api, for one. The changes in this PR are just for the v3 release. The only exception is I added `fulfillment` to the write-files script so that we can publish the fulfillment mapping to S3, but that code doesn't have any purpose in the `v1-node6` branch. So all of this can/should go into the `v3` release for use by by apps on Node14+